File: branches/with-libpq-backend/xml-only/src/items/StandardItem.cpp
/* 
 * File:   StandardItem.cpp
 * Author: jolo
 * 
 * Created on 10. Dezember 2009, 20:09
 */

#include "StandardItem.h"

#include "Dataset.h"

StandardItem::StandardItem(Dataset* d)
: QStandardItem(),
_data(d) {
    setText(_data->getName());
    setToolTip(_data->makeToolTipText());
}

StandardItem::~StandardItem() {
}

File: branches/experimental/py/Gui/ItemModels/StandardItem.py
from PyQt4.QtGui import *

class StandardItem(QStandardItem):
    def __init__(self, data):
        QStandardItem.__init__(self)
        self.data = data
        self.setText(unicode(self.data.name))
        self.setToolTip(unicode(self.data.makeToolTip()))
